pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -40,7 +40,10 @@ func validateInput(c utils.CommandLine) error {
 		version := args.Get(1)
 		_, err := semver.NewVersion(version)
 		if err != nil {
-			logger.Info(color.YellowString("The provided version doesn't use semantic versioning format\n\n"))
+			_, err = semver.NewConstraint(version)
```

Comment:
IMO this is kinda strange behaviour, especially for the CLI. Is this something that has been requested by users / customers?

Reply:
nope, this is from my own, we can live without it but I thought it could be useful. 

Regarding being strange, it's common to support ranges for package managers 🤷  For example:

```bash
$ go get 'github.com/gorilla/mux@<v1.8.0'
. . .
go: downgraded github.com/gorilla/mux v1.8.1 => v1.7.4
```

or 

```bash
$ npm install 'sass@>1.63.2'
added 6 packages, changed 1 package, and audited 1356 packages in 1s
```

Reply:
TIL! Thanks for the extra context.

IMO I'd be inclined to keep things simple unless we need it. I find working with some of these functions are already confusing since version is optional and now if we have to consider it as a constraint too (without typing, still as a string) then there's bound to be a foobar at some point 😅

Reply:
Yeah, I didn't ask too much because I found this fairly simple to implement, so why not? But it's true that it's more code to maintain. Maybe we can have some product feedback @sympatheticmoose WDYT? Have you ever heard of someone complaining that the version choice is too restrictive? (being only latest or a specific one)

Reply:
> Have you ever heard of someone complaining that the version choice is too restrictive?

Not that I've seen. If there's concerns we can definitely do some more discovery. One option might be to run this past the champions group.

Reply:
It's true that Grafana plugins don't support multiple branches so the use case for requiring branches is more uncommon. Let's close this then, we can always reopen it if we receive a request.